.github/actions/generate-coverage/tests/conftest.py

```
@@ -0,0 +1,35 @@
+
+@pytest.fixture
+def shell_stubs(tmp_path, monkeypatch) -> StubManager:
+    """Return a ``StubManager`` configured for the current test."""
+
+    dir_ = tmp_path / "stubs"
+    mgr = StubManager(dir_)
+    import shellstub as mod
+    mod._GLOBAL_MANAGER = mgr
+    monkeypatch.setenv("PATH", f"{dir_}{os.pathsep}{os.getenv('PATH')}")
+    monkeypatch.setenv(
+        "PYTHONPATH", f"{ROOT}{os.pathsep}{os.getenv('PYTHONPATH','')}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)